### ISSUE TITLE
Is this CSS framework still maintained?

### ISSUE THREAD

**Issue by woodcox** [bonsaicss/bonsai.css#79]
Hi, just wondering if this repo is still maintained?

**Comment by DannyBen** [comment]
This CSS framework is the most thoughtful and most well made framework in the classless category (and beyond). I really really like how it is designed and presented. I hope it is still alive.

**Comment by saolof** [comment]
Agreed, this is one of the most beautiful pieces of CSS I've seen and it meets my needs perfectly.

**Comment by NeveIsa** [comment]
Yes, the best.
